x-pack/plugins/security_solution/cypress/integration/timelines/creation.spec.ts

```diff
@@ -39,70 +25,72 @@ import {
   addNotesToTimeline,
   closeTimeline,
   createNewTimeline,
+  goToQueryTab,
   markAsFavorite,
-  openTimelineFromSettings,
   pinFirstEvent,
   populateTimeline,
   waitForTimelineChanges,
 } from '../../tasks/timeline';
-import { openTimeline } from '../../tasks/timelines';
 
 import { OVERVIEW_URL } from '../../urls/navigation';
 
-// Skipped at the moment as there looks to be in-deterministic bugs with the open timeline dialog.
-describe.skip('Timelines', () => {
-  beforeEach(() => {
+describe('Timelines', (): void => {
+  before(() => {
     cleanKibana();
+    loginAndWaitForPage(OVERVIEW_URL);
   });
 
-  it('Creates a timeline', () => {
-    cy.intercept('PATCH', '/api/timeline').as('timeline');
+  describe('Toggle create timeline from plus icon', () => {
+    after(() => {
+      closeTimeline();
+    });
 
-    loginAndWaitForPage(OVERVIEW_URL);
-    openTimelineUsingToggle();
-    addNameAndDescriptionToTimeline(timeline);
+    it('toggle create timeline ', () => {
+      createNewTimeline();
+      cy.get('[data-test-subj="timeline-flyout-header-panel"]').should('be.visible');
```

Comment:
I think you forgot to wrap the  `'[data-test-subj="timeline-flyout-header-panel"]'` value and extract it to it corresponding screen file :)